partition/roles/sonic/README.md

```diff
@@ -15,8 +15,8 @@ Deploys a sonic switch.
 | sonic_mgmtif_gateway                                       |           | If using a fixed management IP, the default gateway for the management interface.                                    |
 | sonic_ip_masquerade                                        |           | Enable ip masquerading on eth0.                                                                                      |
 | sonic_breakouts                                            |           | The breakout configuration for ports. E.g. `dict('Ethernet0'='4x25G')`                                               |
-| sonic_ports                                                |           | Special configuration for ports (mtu, fec, have highest precedence)                                                  |
-| sonic_ports.name                                           |           | The port name.                                                                                                       |
```

Comment:
`sonic_ports` is still used... not as list but as dict. I guess the documentation must be updated?